r/tests/testthat/test-csv.R

```diff
@@ -422,6 +422,25 @@ test_that("Write a CSV file with invalid batch size", {
   )
 })
 
+test_that("Write a CSV with custom NA value", {
+  tbl_out1 <- write_csv_arrow(tbl_no_dates, csv_file, na = "NULL_VALUE")
```

Comment:
Is it worth adding a test that `na = ""` works? I'd imagine that would be a common choice,.